multicluster/controllers/multicluster/core/resourceimport_controller.go

```diff
@@ -40,12 +42,12 @@ import (
 
 const (
 	// cached indexer
-	resImportIndexerByNameKind = "resourceimport.by.namekind"
+	resImportIndexer = "resimp.indexer"
```

Comment:
Nit - what you think about "name.kind" or even "nk".

Reply:
done, rename indexer as `name.kind`